### ISSUE TITLE
Adding Feature / removing feature

### ISSUE THREAD

**Issue by AL-HAYALI** [cgoldberg/py-slideshow#3]
1. You have done a really good job, your code is so clean and arranged, I like it. 
2.I want to remove the feature that zooms to the image and make it stable.
2. Showing the image to the center of the screen
3. showing a text box with each image and it's optional to be able to show the description of some image.
4. Showing the full image without cropping it. got this equation for you:
when the image width is more than screen width then resize the image width to be the same as the screen width, keeping in mind to reduce the height by multiplying the image height by the same number that we multiply the image width with to get the screen width this is some block of cold that may helps:
```if img_w > img_h and img_w > scr_w and img_h > scr_h:
    print("Case 1",img_w, img_h, scr_w, scr_h)
    #determining the image size accouring to the equasion
    # x * img_w = scr_w, then multiply x by the img_h, 
    # so in conclusion we multiply the hight by the same number that we have multiplied the x with
    var = int(img_h*(scr_w/img_w))
    img = ImageTk.PhotoImage(Image.open(path).resize((int(scr_w), var)), Image.ANTIALIAS)
    #print("Scr_w / img_w:", scr_w / img_w, "scr_w:",scr_w, "(scr_w/img_w)*img_h:",img_h*(scr_w/img_w), "scr_w:",scr_w)
    #print("final image width:" ,scr_w, "final image height:",var)

# when the hieght of the image is more than it's width and also more than screen height
elif img_h > img_w and img_h > scr_h:
    print("Case 2")
    img = ImageTk.PhotoImage(Image.open(path).resize((int(scr_w*(scr_h/img_h))), scr_h), Image.ANTIALIAS)
```
